docs/docker.md

```diff
@@ -0,0 +1,8 @@
+# Contenedores
+En la búsqueda me he centrado en contenedores oficiales de docker con golang con la versión que he trabajado y basados en linux, en caso de tener instalado demasiadas cosas por defecto también se contempló la idea de modificar los docker files de la búsqueda. En cuanto al criterio de selección me he centrado en el tamaño de los contenedores ya que había una diferencia muy significativa entre unos y otros, esten basados en linux y las herramientas traen instaladas ya que mientras menos tengan, mejor, ya que la imagen pesará menos.
+
+## **Búsqueda de contenedores:**
+Los contenedores los he buscado en Dockerhub, donde había diversos contenedores para diversas versiones de go en función de las distribuciones, las que más destacaban para la versión que buscaba de go son: [alpine](https://github.com/docker-library/golang/blob/ee5d5d94498b7bd72a5047a2980cf762b0d7236f/1.19/alpine3.16/Dockerfile), [debian 11](https://github.com/docker-library/golang/blob/ee5d5d94498b7bd72a5047a2980cf762b0d7236f/1.19/bullseye/Dockerfile), [windows server](https://github.com/docker-library/golang/blob/ee5d5d94498b7bd72a5047a2980cf762b0d7236f/1.19/windows/windowsservercore-ltsc2022/Dockerfile), [nanoserver](https://github.com/docker-library/golang/blob/ee5d5d94498b7bd72a5047a2980cf762b0d7236f/1.19/windows/nanoserver-1809/Dockerfile), [debian 10](https://github.com/docker-library/golang/blob/6bf11786153a087ebfb866ac9417c4381b30df8f/1.18/buster/Dockerfile), [circleci](https://hub.docker.com/r/circleci/golang), [okteto](https://hub.docker.com/r/okteto/golang), [Bitnami](https://hub.docker.com/r/bitnami/golang), [drud(DDEV)](https://hub.docker.com/r/drud/golang-build-container), [IBM](https://hub.docker.com/r/ibmcom/golang-net-ppc64le), [antrea](https://hub.docker.com/r/antrea/golang) y [almalinux](https://hub.docker.com/r/almalinux/golang)
+
+## **Elección del contenedor:**
+Como se ha comentado anteriormente el criterio de elección ha sido el tamaño del contendor y en este caso ha sido fácil decantarse por el contendor de **alpine** ya que en cuanto al tamaño, el de alpine es algo menos que la mitad del tamaño del contenedor de debian.  Los contenedores de Circleci, Okteto, Drud, Bitnami, IBM, Antrea y Almalinux los he descartado porque o bien están basados en la imagen de debian, que como he mencionado antes es muy pesada, o bien traen herramientas preinstaladas que no interesan en mi proyecto además de aumentar el tamaño del contenedor haciendo que estos sean más pesados que el de Alpine.
```

Comment:
typo
Además, Luis, desde el cariño, tratad de justificarlo un poco mejor. No pongáis la lista y ya está, poned realmente los tamaños de las imágenes. He tenido que irme a Docker Hub para comprobar que efectivamente esta tiene un tamaño de 100 megas, mientras que la de Bitname tiene el doble. También hay unas de alpinelinux, curiosamene, que no has mencionado... pero que también son más grandes. Si ponéis el dato concreto y unos enlaces, me ahorráis trabajo. Que no es que no me fíe, es que es la *due diligence* que tenéis que hacer en estos casos...

Reply:
Para la próxima vez me explicaré mejor, disculpe las molestias :smile: